src/test/Fake.Core.UnitTests/Fake.Core.Target.fs

```diff
@@ -587,6 +587,453 @@ let tests =
       let context = run "SimpleTest"
       Expect.equal true context.HasError "Expected failure"
       Expect.equal 2 context.PreviousTargets.Length "Expected context to contain both targets"  // second one as "skipped"
+
+    // These dependencies are taken from large .NET/COM VS sln. Only the names have been changed.
+    // Printing the running order was taking 3 minutes and ~ 12 GB or RAM using FAKE 5.15
+    Fake.ContextHelper.fakeContextTestCase "Running order resolves quickly when there are numerous targets with numerous dependencies" <| fun _ ->
```

Comment:
Any reason not to use `Fake.ContextHelper.fakeContextTestCaseAssertTime` like in some tests above?

Reply:
No reason, just didn't know about it :)